<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fi">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="hours">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество проработанных
часов за месяц</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addInfo_Tail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в конец</string>
    </property>
   </widget>
   <widget class="QPushButton" name="out">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>298</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод всего</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Табельный номер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addInfo_Head">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в начало</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить на №</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="insertNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить №</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletenumber">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ок</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deleteNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAll">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>298</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить всё</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>82</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тариф</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>82</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>сверхурочно</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ПОИСК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>создать\
сохранить как</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>открыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nam">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="otc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тариф</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tarrif">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>142</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>142</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>кол-во часов</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
